tests/functional/conftest.py

```diff
@@ -88,7 +88,7 @@ def do_lti_launch(post_params, get_params=None, **kwargs):
     return do_lti_launch
 
 
-@pytest.fixture
+@pytest.fixture(autouse=True)
```

Comment:
Default to always intercept all HTTP requests. We can revisit if we find we do want to make them for real but not sure if that is desirable ever.